index.bs

```diff
@@ -396,3 +422,81 @@ Acknowledgements {#ack}
 
 Cure53's [[DOMPURIFY]] is a clear inspiration for the API this document
 describes, as is Internet Explorer's {{window.toStaticHTML()}}.
+
+
+
+Appendix A: Interactions between Sanitizer And Trusted Types {#tt}
+======================================================
+
+<em>This section is not normative.</em>
+
+[[TrustedTypes]] can interact in helpful ways with the Sanitizer API. How to
+best do so will likely depend on the intended use of both. Because the
+Sanitizer API is meant to remove all scripting content, the default is to not
+consider the Sanitizer Trusted Types sensitive.
+
+The predominant use case would be to sanitize strings and use the resulting
+fragments directly, just as without Trusted Types:
+
+```js
+const untrusted_input = "....";
+some_node.AppendChildren(sanitizer.santize(untrusted_input));
+```
+
+For cases where handling of a string is desired, a sanitizing policy can easily
+be created using the `bound` method.
+
+```js
+const my_config = { ... };
+const sanitizing_policy = trustedTypes.createPolicy("apolicyname", {
+  createHTML: new Sanitizer(my_config).bound() });
+
+const untrusted_input = "....";
+some_node.innerHTML = sanitizing_policy.createHTML(untrusted_input);
+```
+
+If a sanitizing policy is installed as the default policy, this would
+auto-sanitize any HTML assignments.
+
+```js
+trustedTypes.createPolicy("default", { createHTML: new Sanitizer().bound() });
```

Comment:
Again, this doesn't make sense to me. Why can't we let _Sanitizer_ return _TrustedHTML_? I don't see a reason why developers have to create a policy when Sanitizer is guaranteed to not have dangerous output?

Reply:
I'm indifferent here. Since the sanitizer can directly produce nodes that don't XSS, many applications won't need a TrustedHTML type (nodes also let you avoid the parser-roundtrip). For those that do need strings, creating a policy is relatively simple (and we can improve that even, e.g. `trustedTypes.createPolicy('name', sanitizerInstance)` or `sanitizerInstance.createPolicy('name')`).

OTOH, we do have `trustedTypes.emptyHTML` because it's useful. And it's not terrible if Sanitizer produced TrustedHTML. Having `trusted-types 'none'; require-trusted-types-for 'script'` guaranteeing no DOM XSS is very compelling.

Reply:
If we require policy creation for Sanitizer API, then it becomes really hard for people to roll out the Perfect Types. Because any dependency could be using `innerHTML` and that'll break Perfect Types (because policy creation is required). On the other hand, if we allow Sanitizer to product TrustedHTML without policy creation (I'm okay with `'sanitizer'` keyword), they will help reduce the breakage.